src/index.js

```diff
@@ -37,7 +37,8 @@ ready(() => {
     app.generateSummary();
 
     // Generate PM tools
-    app.initPM(app);
+    const show_pm_factor_field = browser.storage.local.get('show_pm_factor_field');
+    Promise.all([show_pm_factor_field]).then(res => app.initPM(app, res));
```

Comment:
I suppose `Promise.all([])` is not required in this case, but I was a bit lazy. I suppose `show_pm_factor_field` is already a Promise. right?